### PR TITLE
add installation heading to README.md

### PR DESCRIPTION
**Pull request by unitOrNull**
added an `Installation` heading to `README.md` to have (slightly) more detailed installation instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,37 @@
...
-Or check the Modrinth page: https://modrinth.com/mod/recode
+## Installation
+
+### Modrinth or GitHub releases (recommended)
+
+Click [here](https://modrinth.com/mod/recode) to get to the Modrinth page for the mod. Here you can click on the latest featured version to the left, and then just press the Download button.
+
+Alternatively, click [here](https://github.com/homchom/recode/releases) to get to the latest release of the mod, then navigate to Assets, and there you can find the mod's .jar file.
+
+### Building manually
```
Comment (tk2217) [inline README.md:21]: This is generally not something we want to recommend as in development versions can be very unstable and most normal users should not be guided to building an application from source.

File: README.md [hunk 2]
```diff
@@ -5,12 +5,37 @@
...
+If you want to have the most cutting edge recode experience, with some possibility of bugs, or if you want to contribute, you should run the following commands:
+
+- `git clone https://github.com/homchom/recode/`
+- If you're on Windows, run `gradlew.bat build`, else run `./gradlew build` (This may take a long time.)
+
+If you encounter any errors during this process, feel free to ask for help in the Discord server.
+
+After you're done you should find the mod's .jar file in `build/libs/` as `recode-[version number].jar`
+
+### GitHub actions
```
Comment (tk2217) [inline README.md:32]: Again, probably unstable and likely better to wait for a beta.

File: README.md [hunk 3]
```diff
@@ -5,12 +5,37 @@
...
 recode is a utility-focused fabric mod for [DiamondFire](https://mcdiamondfire.com/home/) to make development more fun. It is the official successor to [CodeUtilities](https://github.com/CodeUtilities/CodeUtilities-2.0), from which it was cloned.
 
-You can find the mod jar [here](https://modrinth.com/mod/recode/versions). Future plans include removing bloat/unnecessary features, improving existing code, adding new features, and migrating to Kotlin.
+Future plans include removing bloat/unnecessary features, improving existing code, adding new features, and migrating to Kotlin.
 
 Feel free to join the recode Discord server: https://discord.gg/GWxWtcwA2C
 
 <!-- modrinth_exclude.start -->
-Or check the Modrinth page: https://modrinth.com/mod/recode
+## Installation
```
Comment (tk2217) [inline README.md:13]: While putting this in its own area and not having it incorporated into another paragraph is good, we should probably just direct to modrinth as it's more user friendly and I believe it's auto updated when a release is made on GitHub.

**Review by tk2217 — CHANGES_REQUESTED** [review]

**Comment by unitOrNull** [comment]
after reading tk's comments, this pr is probably not very useful and as they pointed out, it would probably be best to just point them to the modrinth page, so i will be closing this pr

**Pull request closed without merging**